web/src/app/users/UserContactMethodCreateDialog.tsx

```diff
@@ -28,13 +34,19 @@ const userConflictQuery = gql`
   }
 `
 
-export default function UserContactMethodCreateDialog(props) {
+export default function UserContactMethodCreateDialog(props: {
+  userID: string
+  onClose: (contactMethodID?: string) => void
+  disclaimer: string
+  title: string
+  subtitle: string
+}): JSX.Element {
   const [allowSV, allowE, allowW] = useConfigValue(
     'Twilio.Enable',
     'SMTP.Enable',
     'Webhook.Enable',
   )
-  let typeVal = ''
+  let typeVal: ContactMethodType = 'SMS'
```

Comment:
I think voice should be the default since the other 3 out of the 4 types are covered below

```suggestion
  let typeVal: ContactMethodType = 'VOICE'
```